manage-mysql-user.py

```diff
@@ -424,12 +433,21 @@ def handler(event, context):
         mysql_master_password_source,
         mysql_master_password_source_type,
     )
-    execute_statement(
-        "GRANT ALL ON `{}`.* to '{}'@'%';".format(database, mysql_user_username),
-        mysql_master_username,
-        mysql_master_password_source,
-        mysql_master_password_source_type,
-    )
+
+    if "privileges" in event.keys():
+        if len(event["privileges"]) > 0:
+            privileges = event["privileges"]
+            logger.info(
+                f"Granting {privileges} privileges to MySQL user {mysql_user_username}"
+            )
+            execute_statement(
+                "GRANT {} ON `{}`.* to '{}'@'%';".format(
+                    privileges, database, mysql_user_username
+                ),
+                mysql_master_username,
+                mysql_master_password_source,
+                mysql_master_password_source_type,
+            )
```

Comment:
might be worth a log to state "no privileges granted"

Reply:
Good call, done